guardrails/telemetry/runner_tracing.py

```diff
@@ -7,6 +7,7 @@
     Iterator,
     Optional,
 )
+from openinference.semconv.trace import SpanAttributes
```

Comment:
Currently, none of the openinference packages are direct dependencies of guardrails-ai, nor should they be.  Instead this should be handled conditionally in the case that `openinference-semantic-conventions` is installed as a peer dependency.  The same applies to all files where this is imported and used.
```suggestion
try:
    from openinference.semconv.trace import SpanAttributes
except ImportError:
    SpanAttributes = None
```